extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -311,15 +316,18 @@ public static function checkUserCreateTables( $updater ) {
 	 */
 	public static function checkUserParserTestTables( &$tables ) {
 		$tables[] = 'cu_changes';
+
 		return true;
 	}
 
 	/**
 	 * Add a link to Special:CheckUser on Special:Contributions/<username> for
 	 * privileged users.
-	 * @param $id Integer: user ID
-	 * @param $nt Title: user page title
+	 *
+	 * @param $id    Integer: user ID
+	 * @param $nt    Title: user page title
 	 * @param $links Array: tool links
+	 *
 	 * @return true
 	 */
 	public static function loadCheckUserLink( $id, $nt, &$links ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove the unused function parameter "$id". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)